src/content/ui/navigation/deep-linking.md

```diff
@@ -60,9 +60,9 @@ To get started, see our cookbooks for Android and iOS:
 If you have written a plugin to handle deep links, as described in
 [Deep Links and Flutter applications][plugin-linking]
 (a free article on Medium),
-it will continue to work until you opt in to this behavior by adding
-`FlutterDeepLinkingEnabled` to `Info.plist` or
-`flutter_deeplinking_enabled` to `AndroidManifest.xml`, respectively.
+You should opt out the default flutter deep links handler by adding
+`FlutterDeepLinkingEnabled` to false in `Info.plist` or
+`flutter_deeplinking_enabled` to false in `AndroidManifest.xml`, respectively.
```

Comment:
```suggestion
you should opt out the Flutter's default deep link handler.
To do this, set `FlutterDeepLinkingEnabled` to false in `Info.plist` _or_
`flutter_deeplinking_enabled` to false in `AndroidManifest.xml`.
```

Just double checking. You don't need to do both? It's one or the other?

Reply:
One for iOS and one for Android,  

You don't need to do both? It's one or the other? >>It depends on if you develop on both platform or just one.